<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EngTools</string>
  </property>
  <widget class="QPushButton" name="bt_runHKP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>run HKP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_runDTP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>run DTP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stsHKP">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>GOOD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_stsDTP">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>GOOD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_inst_simul">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Instrument Simulation</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_real">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Real</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
